docs/patterns/components/Alert/index.css

```diff
@@ -0,0 +1,13 @@
+@media only screen and (max-width: 375px) {
```

Comment:
@scurker @thuey Yeah, definitely think it would be good to talk about/bring up at the design review. There seem to be a fair amount of reflow issues at 320px and I think it would be helpful to have some standard breakpoints for handling things like using grids/flexbox, as changing flex direction/ordering based on breakpoints is one of the best ways to change things like direction and ordering. Perhaps maybe even benefit from having a `<Flex>` and/or `<Grid>` component that manages that handles things.

Reply:
I mostly agree with you, especially about having some standards. I will say that I personally rarely change flex direction, though. In practice, I have found flex-wrap: wrap` without a breakpoint is typically sufficient.